### PR TITLE
Add Uniswap V3 into GitHub Worflows

### PR DESCRIPTION
**Pull request by lyubenkov**

### CODE REVIEW

File: .github/workflows/dapps.yml [hunk 1]
```diff
@@ -115,6 +115,49 @@ jobs:
...
+      run: |
+        CWD=/tmp/uniswap-${{ github.run_id }}
+        mkdir -p ${CWD}
+        git clone https://github.com/neonlabsorg/Uniswap-V3-NEON.git ${CWD}
+        docker build -t uniswap:latest ${CWD}
+        proxy_url=http://${{ env.PROXY_IP }}:9090/solana
+        test -t 1 && USE_TTY="-t"
+        docker run -i ${USE_TTY} --name=uniswap-${{ github.run_id }} -e NEON_PROXY_URL=${proxy_url} -e NEON_ACCOUNTS=${{ env.ACCOUNTS }} --entrypoint /bin/bash uniswap:latest -c "npx hardhat run /usr/src/app/scripts/deploy.ts"
+        docker cp uniswap-${{ github.run_id }}:/usr/src/app/report.json report.json
+        docker rm -f uniswap-${{ github.run_id }}
```
Comment (gigimon) [inline .github/workflows/dapps.yml]: rename uniswap-report to uniswapv3-report

File: .github/workflows/dapps.yml
Comment (kristinaNikolaevaa) [inline .github/workflows/dapps.yml:1]: You need fix ALL_DAPPS env value (at the begin on the file somewhere)

Comment (lyubenkov) [reply .github/workflows/dapps.yml:1]: Fixed, thanks!

File: .github/workflows/dapps.yml (hunk 1 above)
Comment (lyubenkov) [reply .github/workflows/dapps.yml]: Renamed reports and added versions here in docker commands. Thanks!

**Review by gigimon — APPROVED** [review]

**Review by gigimon — DISMISSED** [review]
Please check our develop branch and remove docker build for dapp.

**Review by kristinaNikolaevaa — APPROVED** [review]

**Review by gigimon — APPROVED** [review]

**Pull request merged**